.travis.yml

```diff
@@ -136,24 +121,26 @@ jobs:
         - sourceline: 'deb [arch=amd64] https://packages.microsoft.com/ubuntu/14.04/prod trusty main'
           key_url: https://packages.microsoft.com/keys/microsoft.asc
         - sourceline: 'ppa:maarten-fonville/protobuf'
+        - sourceline: 'ppa:fsgmhoward/shadowsocks-libev'        
         packages:
-        - gcc
+        - binutils-dev
+        - build-essential
+        - cmake
         - g++
-        - libssl-dev
+        - gcc
         - libcurl4-openssl-dev
-        - libelf-dev
         - libdw-dev
-        - binutils-dev
+        - libelf-dev
         - libiberty-dev
-        - libsnappy-dev
+        - libprotobuf-dev
         - librocksdb
-        - powershell
-        - cmake
-        - zlib1g-dev
-        - build-essential
+        - libsnappy-dev
+        - libsodium-dev        
+        - libssl-dev
         - pkg-config
+        - powershell
         - protobuf-compiler
-        - libprotobuf-dev
+        - zlib1g-dev
```

Comment:
Please remind me, why do tests have their own package dependency list? It seems to be more or less the same as the global one.

Reply:
Test stage contains additional `powershell` package.